os/net/lwip/src/include/lwip/dhcp.h

```diff
@@ -152,6 +157,15 @@ void dhcp_stop(struct netif *netif);
 
 #if LWIP_DHCP_HOSTNAME
 void dhcp_hostname(struct netif *netif, char *name);
+/**
+ * @brief dhcp_sethostname() set hotsname of dhcpc
+ *
+ * @internal
+ * @param[in] msg contains netif and hostname
```

Comment:
Please check the comment, there are two params, `netif` and `arg`.

Reply:
Thanks I'll modify it